API/plumber.R

```diff
@@ -2,14 +2,15 @@ library(plumber)
 library(pins)
 library(tibble)
 library(xgboost)
-library(bikeHelpR)
 library(lubridate)
 library(dplyr)
+library(tidyr)
+library(tidymodels)
 
-pins::board_register_rsconnect(server = "https://colorado.rstudio.com/rsc",
+board_register_rsconnect(server = "https://colorado.rstudio.com/rsc",
                                key = Sys.getenv("RSTUDIOCONNECT_API_KEY"))
-mods <- list(r_xgb = pins::pin_get("alex.gold/bike_model_rxgb", board = "rsconnect"))
-stats <- pins::pin_get("alex.gold/bike_station_info", board = "rsconnect")
+model_details <- pin_get("bike_model_rxgb", board = "rsconnect")
```

Comment:
This was originally a list since I was thinking we'd end up doing model comparison within the app. Never got there. Not sure if it's worth building out or not...

Reply:
I opted to simplify the code until the day comes when we are doing model comparison, which I don't think is critical for the MVP of the demo